types/redux-devtools/redux-devtools-tests.tsx

```diff
@@ -12,10 +12,12 @@ const DevTools = createDevTools(
   <DevToolsMonitor />
 )
 
-const finalCreateStore = compose(
+const storeEnhancer = compose(
   DevTools.instrument(),
   persistState('test-session')
-)(createStore)
+) as GenericStoreEnhancer
```

Comment:
For more information about this change, see here: https://github.com/reactjs/redux/issues/1937